File: ruby/small-problems/easy8/e06.rb
def fizzbuzz_value(n)
  result = ""
  result += "Fizz" if n%3 == 0
  result += "Buzz" if n%5 == 0
  result == "" ? n.to_s : result
end

def fizzbuzz(start, ending)
  start.upto(ending) do |n|
    result = fizzbuzz_value(n)
    result += ", " unless n == ending
    print result
  end
  puts
end

fizzbuzz(1,15)

File: javascript/small-problems/easy5/e06.js
function sequence(n) {
  let result = [];
  for(let i = 1; i <= n; i ++) {
    result.push(i);
  }
  console.log(result);
}

sequence(5);    // [1, 2, 3, 4, 5]
sequence(3);    // [1, 2, 3]
sequence(1);    // [1]
